rubocop-3.3.yml

```diff
@@ -0,0 +1,5 @@
+inherit_from:
+  - core-settings.yml
```

Comment:
After seeing this i wonder if we should just inherit from this file and set the ruby version in the repos? then in the future we dont have to change this repo. So in gambit we could change it to be:
```
inherit_from:
  - https://raw.githubusercontent.com/CiscoSecurityServices/rubocop/master/core-settings.yml

AllCops:
  TargetRubyVersion: 3.3
```